docs/configuration.md

```diff
@@ -154,3 +167,17 @@ default: 64
 ```
 
 Maximum number of jobs to run concurrently.
+
+## Limit overrides
+
+```yaml
+key: PISTON_LIMIT_OVERRIDES
+default: {}
+```
+
+Per-language overrides/exceptions for the each of `max_process_count`, `max_open_files`, `max_file_size`,
+`compile_memory_limit`, `run_memory_limit`, `compile_timeout`, `run_timeout`, `output_max_size`. Defined as follows:
+```
+PISTON_LIMIT_OVERRIDES={"c++":{"max_process_count":128}}
```

Comment:
does this not cause a quoting issue?

Reply:
It only breaks when the JSON is surrounded by single quotes. Both of these will work:
```
'PISTON_LIMIT_OVERRIDES={ "csharp": { "max_process_count" : 128 }}'
```
(entire variable is surrounded with single quotes, spaces are ok)
```
PISTON_LIMIT_OVERRIDES={"csharp":{"max_process_count":128}}
```
(no single quotes, no spaces)

Reply:
![image](https://user-images.githubusercontent.com/11236933/136195196-cb9e76b5-d369-464c-9b0d-51cda0e97a7b.png)
will this not happen?